backend/controllers/attendanceStatistics.js

```diff
@@ -0,0 +1,106 @@
+const moment = require('moment')
+const { merge, switchDateFormat, getCurrentShift, pascalToString } = require('../utils')
+
+const getReasonCountMap = (data, reasons) => {
+  const countAttendancesForReason = reason =>
+    data.filter(
+      attendance =>
+        attendance && attendance.absentReason && attendance.absentReason.toLowerCase() === reason.toLowerCase()
```

Comment:
don't change it in this PR, but all of the `.toLowerCase()` looks to be pointless since the data is all coming in same format anyway.